### PR TITLE
Add KDE Conference

### PR DESCRIPTION
**Pull request by championshuttler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ If you would like to be notified of future conferences that would be added to th
 | [LaravelLive India](https://laravellive.in/) | 19 Mar | New Delhi | A PHP Laravel Conference. |
 | [PyDelhi Conf](https://conference.pydelhi.org/) | 18-19 Mar | IIM Lucknow Noida Campus, Noida | PyDelhi conference is hosted annually by PyDelhi Community with an aim to promote Python programming language. |
 | [FOSSMeet](http://fossmeet.in/2017/) | 10-12 Mar | NIT Calicut | FOSSMeet is an annual event on Free and Open source software, conducted at National Institute of Technology, Calicut. |
+|[KDE Conference](https://conf.kde.in/)| 10-12 Mar | IIT Guwahati | It is a platform for Qt/KDE contributors, developers and enthusiasts to come together, share their knowledge. |
```
Comment (nikhita) [inline README.md]: Can you update the venue as `IIT Guwahati, Assam`?

Comment (nikhita) [reply README.md]: `enthusiasts to come together, share their knowledge.` -> `enthusiasts to come together and share their knowledge.`

**Comment by nikhita** [comment]
Also, please squash the commit to a single commit. :)

File: README.md (hunk 1 above)
Comment (championshuttler) [reply README.md]: yeah sure :)

**Comment by championshuttler** [comment]
i am about to ask you that how to squash commits and you did already :P

**Comment by nikhita** [comment]
@championshuttler Github provides a way to squash and merge in their UI and I used that. However, it doesn't create a merge commit so I don't prefer that usually. :)

In general, you can read about how to squash commits here: https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/.

**Pull request merged**